### DISCUSSION TITLE
side bar with page preview

### DISCUSSION THREAD

**Discussion by hpdonat** [sumatrapdfreader/sumatrapdf#4238] [Q&A]
Is there a view option or any setting in Sumatra, for a side bar, that has a page by page preview, that you can scroll through, as in most other pdf viewers? So, instead of having to scroll through each page in entirety, I can simply scroll through thumbnails of each page, on the left side of the document.

**Comment by GitHubRulesOK** [comment]
Nothing at present and several  issue/requests opened long ago. (even got fairly heated)
It is not a trivial exercise and an associated heated discussion often arises over right hand scroll bar visual progress.

**Reply by hpdonat** [reply]
I wonder why this gets people worked up?

**Reply by GitHubRulesOK** [reply]
> wonder why this gets people worked up?  

IT is a popular feature in many other heavier readers such as Acrobat look alike's and some browsers and as discussed there does not have to be a significant slow down plus SumatraPDF has a previewer that does much the same when installed using explorer here we see the Acrobat preview of page 3 on the right ! Way faster than using Acrobat itself, hence I use it a lot.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7dd1a974-a488-416e-afb7-0c1c63752fd6)
